### PR TITLE
Story/ccp 117

### PR DESCRIPTION
**Pull request by mkarthik90**
[CCP-117] Fixed the code review comments

### CODE REVIEW

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/bean/ConceptEditBean.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	public List getConceptEntryList() {
+		return conceptEntryList;
+	}
+
+	public void setConceptEntryList(List conceptEntryList) {
+		this.conceptEntryList = conceptEntryList;
+	}
+
+	public Map<String, String> getPosMap() {
+		posMap.put(edu.asu.conceptpower.core.POS.NOUN, POS.NOUN);
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/bean/ConceptEditBean.java]: why do you use the full path here? is it a different POS than the other one?

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/bean/ConceptEditBean.java [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+		posMap.put(edu.asu.conceptpower.core.POS.ADVERB, POS.ADVERB);
+		posMap.put(edu.asu.conceptpower.core.POS.ADJECTIVE, POS.ADJECTIVE);
+		return posMap;
+	}
+
+	public void setPosMap(Map<String, String> posMap) {
+		this.posMap = posMap;
+	}
+
+}
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/bean/ConceptEditBean.java]: formatting

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptEditController.java [hunk 3]
```diff
@@ -216,13 +184,71 @@ public String confirmlEdit(@PathVariable("id") String id,
...
+			}
+
+			jsonStringBuilder.append("{");
+			jsonStringBuilder.append("\"Id\":\"" + syn.getId() + "\"");
+			jsonStringBuilder.append(",");
+			jsonStringBuilder.append("\"Word\":\"" + syn.getWord() + "\"");
+			jsonStringBuilder.append(",");
+			jsonStringBuilder.append("\"Description\":\"" + syn.getDescription().replaceAll("\"", "'") + "\"");
+
+			if (posRequired == true) {
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptEditController.java]: there is not really a reason to sometimes not have the POS; even if you don't show it (which is unlikely) you can add it here; would be easier and cleaner.

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptEditView.jsp [hunk 4]
```diff
@@ -4,248 +4,248 @@
...
-						$("#addedSynonyms").html(html);
+                        var htmlValue = '<tr id='+eachSynonym.Id+'><td align="justify"><font size="2">'
+                                + '<a onclick="synonymTemporaryRemove(\''
+                                + eachSynonym.Id
+                                + '\')">Remove</a>'
+                                + '</font></td>';
+                        htmlValue += '<td align="justify"><font size="2">'
+                                + eachSynonym.Word + '</font></td>';
+                        htmlValue += '<td align="justify"><font size="2">'
+                                + eachSynonym.Description + '</font></td></tr>';
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptEditView.jsp]: You might want to think about refacting this to use jquery's append method.

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptEditView.jsp [hunk 5]
```diff
@@ -4,248 +4,248 @@
...
+                                                    + '</font></td></tr>';
+                                            // arr.push(eachSynonym.SynonymObject);
 
-									}
-								});
-					});
-</script>
+                                        }
+                                        // document.getElementById("conceptEntryList").value = arr;
+                                        html += '</tbody></table>';
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptEditView.jsp]: same here

**Comment by jdamerow** [comment]
see comments

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptEditView.jsp (hunk 4 above)
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptEditView.jsp]: @jdamerow  Should I refactor htmlvalue? I am not sure which variable I need to refactor.

Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptEditView.jsp]: looking at this again, I think I confused it with bootstrapTable append. You already do what I had in mind. We might want to think to switch to Bootstrap for the layout, and then we can make this much prettier. but for now, let's leave it with this.

**Pull request closed without merging**